### PR TITLE
python: add options control command

### PR DESCRIPTION
**Pull request by jelly**
The Cockpit terminal uses options to set the rows and cols.

### CODE REVIEW

File: src/cockpit/channel.py [hunk 1]
```diff
@@ -95,6 +97,9 @@ def do_close(self):
     def do_pong(self, message):
         pass
 
+    def do_options(self, message):
+        pass
```
Comment (allisonkarlitskaya) [inline src/cockpit/channel.py:101]: how about `raise ChannelError("this channel don't do options")`?

Comment (jelly) [reply src/cockpit/channel.py]: Sounds fine!

File: src/cockpit/channel.py [hunk 2]
```diff
@@ -95,6 +97,9 @@ def do_close(self):
     def do_pong(self, message):
         pass
 
+    def do_options(self, message):
+        raise ChannelError('This channel does not implement "options"')
```
Comment (allisonkarlitskaya) [inline src/cockpit/channel.py:101]: It needs a problem code as the first field.  Probably `protocol-error`?  What does the C bridge do here, anyway?

Comment (jelly) [reply src/cockpit/channel.py:101]: Not, `not-supported`?

**Review by allisonkarlitskaya — APPROVED** [review]

**Pull request merged**